Update README.md (Removed references to NMS) (#114)
nginxinc/ansible-role-nginx-management-suite: README.md

[![Molecule CI/CD](https://github.com/nginxinc/ansible-role-nginx-management-suite/workflows/Molecule%20CI/CD/badge.svg)](https://github.com/nginxinc/ansible-role-nginx-management-suite/actions/workflows/molecule.yml/badge.svg)
[![License](https://img.shields.io/badge/License-Apache--2.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)

> # Ansible NGINX Instance Manager (NIM) Role

Reviewer: ```suggestion
# Ansible NGINX Instance Manager Role
```

Reviewer: FYI: Marketing says we can't abbreviate product names, unfortuantly. They need to be spelled out each time.

This role only installs NGINX Instance Manager.

**Note:** This role is still in active development. There may be unidentified issues and the role variables may change as development continues.

## Requirements

1. NGINX Instance Manager License Files
2. [NGINX Ansible Role (**nginxinc.nginx**)](https://github.com/nginxinc/ansible-role-nginx)

### NGINX Instance Manager Certificate Files

> Installing NIM requires the NIM certificate files to access the repository. Log in to [MyF5](https://account.f5.com/myf5) or follow the link in the trial activation email to download the NIM repo **.crt** and **.key** files:

Reviewer: ```suggestion
Installing NGINX Instance Manger requires the Instance Manager certificate files to access the repository. Log in to [MyF5](https://account.f5.com/myf5) or follow the link in the trial activation email to download the NIM repo **.crt** and **.key** files:
```

### NGINX Instance

> NIM requires an NGINX instance, either NGINX OSS or NGINX Plus as a frontend only. This role handles this by defining a dependency to the [NGINX Ansible Role](https://github.com/nginxinc/ansible-role-nginx), named **nginxinc.nginx**. Because of this dependance, you can set variables related to **nginxinc.nginx** when using this role. For example, `nginx_type` is an **nginxinc.nginx** variable that can be [set like how you would any other Ansible variable](https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_variables.html#where-to-set-variables). So if your playbook defines `nginx_type: plus`, this NIM role will call the **nginxinc.nginx** role which will install NGINX Plus. Refer to the [Ansible Role NGINX](https://github.com/nginxinc/ansible-role-nginx) for more details.

Reviewer: ```suggestion
NGINX Instance Manager requires an NGINX instance, either NGINX OSS or NGINX Plus as a frontend only. This role handles this by defining a dependency to the [NGINX Ansible Role](https://github.com/nginxinc/ansible-role-nginx), named **nginxinc.nginx**. Because of this dependance, you can set variables related to **nginxinc.nginx** when using this role. For example, `nginx_type` is an **nginxinc.nginx** variable that can be [set like how you would any other Ansible variable](https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_variables.html#where-to-set-variables). So if your playbook defines `nginx_type: plus`, this NIM role will call the **nginxinc.nginx** role which will install NGINX Plus. Refer to the [Ansible Role NGINX](https://github.com/nginxinc/ansible-role-nginx) for more details.
```

* Molecule is used to test the various functionalities of the role. The recommended version of Molecule to test this role is `4.0.1`.
* Instructions on how to install Molecule can be found in the [Molecule website](https://molecule.readthedocs.io/en/latest/installation.html). *You will also need to install the Molecule Docker driver.*

> * To run the Molecule tests, you must copy your NIM license to the role's [`files/license`](https://github.com/nginxinc/ansible-role-nginx-management-suite/blob/main/files/license/) folder.

Reviewer: ```suggestion
* To run the Molecule tests, you must copy your NGIXN Instance Manager license to the role's [`files/license`](https://github.com/nginxinc/ansible-role-nginx-management-suite/blob/main/files/license/) folder.
```

## Usage

> Take these steps in order to install NGINX Instance Manager (NIM) using this ansible role.

Reviewer: ```suggestion
Take these steps in order to install NGINX Instance Manager using this ansible role.
```

### Create Playbook

> Create a playbook file, `nms-install.yml`, using the following example. Here, we are installing NIM with NGINX Plus.

Reviewer: ```suggestion
Create a playbook file, `nms-install.yml`, using the following example. Here, we are installing NGINX Instance Manager with NGINX Plus.
```

          key: nginx-repo.key
```

> ### Install NGINX Instance Manager (NIM)

Reviewer: ```suggestion
### Install NGINX Instance Manager
```

> Run the following command to run the playbook which will install NIM.

Reviewer: ```suggestion
Run the following command to run the playbook which will install NGINX Instance Manager.
```

```shell
ansible-playbook -i inventory nms-adm-install.yml
```

> ## Using Latest Edge of NIM Ansible Role, aka the `main` Branch

Reviewer: ```suggestion
## Using Latest Edge of NGINX Instance Manager Ansible Role, aka the `main` Branch
```
